.env.example

```
@@ -60,3 +60,12 @@ MOCK_INVOICE_PAYMENTS=false
 # ============================================
 
 REDIS_URL=redis://localhost:6379
+
+# ============================================
+# AI Accountant Service (Optional)
+# AI-powered accounting for automated journal entries & reports
+# ============================================
+
+AI_ACCOUNTANT_URL=http://localhost:8000
+# Generate a strong API key: openssl rand -hex 32
+AI_ACCOUNTANT_API_KEY=your_ai_accountant_api_key_here
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AI_ACCOUNTANT_API_KEY key should go before the AI_ACCOUNTANT_URL key

(UnorderedKey)